### ISSUE TITLE
第31行的onehotlabel没有必要再乘一次吧。原则上来说这个只要结果乘一次就好啦

### ISSUE THREAD

**Issue by hanghang2333** [zhezh/focalloss#1]

**Comment by hushunda** [comment]
的确没必要，这个程序写的啰说了，还没有softmax

**Comment by alyato** [comment]
hi,我用keras，backend 用TensorFlow。
请问，我用这个code，要的是下面的这个么

> func_fl = focal_loss(labels, model_out, fl_gamma, fl_alpha)

还是这一行
> loss = tf.reduce_mean(func_fl)

谢谢。

**Comment by lilmangolil** [comment]
您好，想请问一下分类所用损失函数是tf.nn.sparse_softmax_cross_entropy_with_logits，可否直接将这个函数换做focal-loss，其中的label和logits直接用呢？谢谢~
